installer/roles/kubernetes/templates/postgresql-values.yml.j2

```diff
@@ -40,10 +40,20 @@ image:
   registry: {{ pg_image_registry }}
 {% endif %}
 {% if pg_image_registry is not defined %}
-  registry: docker.io/bitnami
+  registry: docker.io
```

Comment:
I agree with the code and the pattern. LGTM.

Nit: I would have an esthetic comment:

```
  registry: '{{ pg_image_registry is defined | ternary(pg_image_registry, "docker.io") }}'
  repository: '{{ pg_image_repository is defined | ternary(pg_image_repository, "bitnami/postgresql") }}'
  tag: '{{ pg_image_version is defined | ternary(pg_imave_version, "12.5.0") }}'
  ```
  
  Not a blocker, just a feedback

Reply:
@Spredzy thanks for your comment. I agree and my plans were to create a dedicated PR to simplify this `j2` file, but if you prefer I can add it to this one.